ONSdigital/census-rh-ui
# Cr 1022 updates to individual request

# Motivation and Context
Updates to improve text and sign posting of individual journeys

# What has changed

- Added function to call campaign site links in centrally managed way

- Removed generic Request Code Intro page functionality and replaced with specific Request Individual code intro page as it is the only one that will be used

- Added post method to Individual intro page to allow for session detection

- Various template updates to add trailing removal markers and collapse error messaging

- Added new 'I don't have a fixed address' collapsible

- Updated expired template to update in page link

- Removed 'new' from individual request text throughout

- Added panel to individual postal journey to declare what the envelope will look like

- Updated so that if user selects to request an individual form at the select method point in the household journey they are presented with the Individual intro screen before continuing, but as an individual.

- Updated unit tests, including adding step to cope with post on individual intro throughout individual tests

# Cucumber update now required for updated error message
https://github.com/ONSdigital/census-rh-cucumber/pull/97

# How to test?
Follow individual UAC request path from /start/, see revised intro apge and marking throughout
Start a request household code, but select to request an individual on the select method of delivery page, and switch to the individual journey without selecting the address again

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1022
https://github.com/ONSdigital/census-rh-cucumber/pull/97

# Screenshots (if appropriate):
